ios/MusicControlManager.m

```diff
@@ -9,21 +9,56 @@
 @interface MusicControlManager ()
 
 @property (nonatomic, copy) NSString *artworkUrl;
+@property (nonatomic, strong) NSDictionary *mediaKeys;
 
 @end
 
+NSString * const MEDIA_PLAYBACK_RATE = @"playbackRate";
+
 @implementation MusicControlManager
 
 @synthesize bridge = _bridge;
 
 RCT_EXPORT_MODULE()
 
+- (NSDictionary *) mediaKeys {
+    if (_mediaKeys == nil) {
+        _mediaKeys = @{
+                @"album": MPMediaItemPropertyAlbumTitle,
+                @"trackCount": MPMediaItemPropertyAlbumTrackCount,
+                @"trackNumber": MPMediaItemPropertyAlbumTrackNumber,
+                @"artist": MPMediaItemPropertyArtist,
+                @"composer": MPMediaItemPropertyComposer,
+                @"discCount": MPMediaItemPropertyDiscCount,
+                @"discNumber": MPMediaItemPropertyDiscNumber,
+                @"genre": MPMediaItemPropertyGenre,
+                @"persistentID": MPMediaItemPropertyPersistentID,
+                @"duration": MPMediaItemPropertyPlaybackDuration,
+                @"title": MPMediaItemPropertyTitle,
+                @"elapsedPlaybackTime": MPNowPlayingInfoPropertyElapsedPlaybackTime,
+                @"playbackRate": MPNowPlayingInfoPropertyPlaybackRate,
```

Comment:
I don't know if this is supposed to work like this, but in Android these properties are called `speed` and `elapsedTime` instead of `playbackRate` and `elapsedPlaybackTime`.
Also, where is the `state`? Isn't it supposed to replace `togglePause`?